pkgs/top-level/all-packages.nix

```diff
@@ -5711,7 +5711,7 @@ in
   padthv1 = libsForQt5.callPackage ../applications/audio/padthv1 { };
 
   page = callPackage ../tools/misc/page { };
-  
```

Comment:
Oh, Kate automatically remove spaces from here... I can revert this.

Reply:
No, it's fine. Trailings spaces shouldn't have been there in the first place